### ISSUE TITLE
1.1.0 coerces all code to Strict Mode

### ISSUE THREAD

**Issue by schmod** [olov/ng-annotate#215]
As of v1.1.0, the following code now causes ngAnnotate to fail:

```js
function foo() {
  delete Bar;
};
```

```
$ ng-annotate foo.js --add
error: couldn't process source due to parse error
Deleting local variable in strict mode (3:2)
```

Note that the input is *not* in strict mode.  

This regression appears to have been introduced in d9610a913589235a63712d7771f9ba27c7c6f3e5.  Acorn appears to be behaving correctly, as modules are [implicitly always in strict mode](http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code).  

This is an issue for us, because we run all of our JS (including external libraries) through ng-annotate, which has proven to be safe up until now.  Notably, Raphael contains some "non-strict" code that causes Acorn to throw an exception when ngAnnotate tries to parse it.

**Comment by olov** [comment]
Ouch - very sorry about that! Just released 1.1.1 which fixes the problem.

**Comment by schmod** [comment]
awesome.  thanks for the quick fix!!!
